front/types/chat.ts

```diff
@@ -1,3 +1,5 @@
+import { TimeRangeId } from "@app/components/use/ChatTimeRangePicker";
```

Comment:
Nope :)

Def not want to have types imported from a component in here. The idea of these types is to be our source of truth for types that go through the network interface.

Def want to move the TimeRangeId here instaed of the other way around